spec/factories/document_factory.rb

```diff
@@ -4,6 +4,8 @@
   factory :document do
     content_id { SecureRandom.uuid }
     locale { I18n.available_locales.sample }
+    base_path { "/#{SecureRandom.alphanumeric(8)}" }
```

Comment:
What would you think about only specifying things that are necessary to create a valid record in the factories?

Reply:
Fine with me :-) Updated.